src/Encryption.php

```diff
@@ -366,14 +367,14 @@ private static function convertBase64ToBigInteger(string $value): BigInteger
     {
         $value = unpack('H*', Base64Url::decode($value));
 
-        return BigInteger::fromBase($value[1], 16);
+        return BigInteger::fromBase($value !== false ? $value[1] : '', 16);
```

Comment:
Maybe instead throw a readable error if unpacking has failed?

Reply:
Sure, will have a look at both of these.

Reply:
Added an exception. I hope the error message is clear enough.